packages/desktop-gui/src/project-nav/browsers.jsx

```diff
@@ -30,16 +31,16 @@ export default class Browsers extends Component {
   }
 
   _closeBrowserBtn () {
-    if (this.props.project.browserState === 'opened') {
-      return (
-        <li className='close-browser'>
-          <button className='btn btn-xs btn-danger' onClick={this._closeBrowser.bind(this)}>
-            <i className='fa fa-fw fa-times'></i>
-            Stop
-          </button>
-        </li>
-      )
-    }
+    if (!this.props.project.isBrowserState(Project.BROWSER_OPEN)) return null
+
+    return (
+      <li className='close-browser'>
+        <button className='btn btn-xs btn-danger' onClick={this._closeBrowser.bind(this)}>
+          <i className='fa fa-fw fa-times'></i>
```

Comment:
Since this is purely visual and there is already content text: `"Stop"`, we should hide this from screen readers altogether. Possibly using something like `aria-hidden` attribute.